include/license.php

```diff
@@ -91,8 +91,7 @@ public function __construct( $file, $item_name, $version, $author, $api_url = nu
 		}
 
 		// Updater
-		add_action( 'admin_init', array( $this, 'auto_updater' ), 0 );
-		add_action( 'cli_init', array( $this, 'auto_updater' ), 0 ); // For WP CLI.
+		add_action( 'plugins_loaded', array( $this, 'auto_updater' ), 10 );
```

Comment:
Do we really need to use an action here? As far as I know, at this point we are already hooked to `plugins_loaded` (with priority 1).
```suggestion
		$this->auto_updater();
```

Reply:
You're right, let's simplify!